core/src/main/java/io/grpc/ServerInterceptors.java

```diff
@@ -179,101 +173,96 @@ public InputStream parse(final InputStream stream) {
   public static <T> ServerServiceDefinition useMarshalledMessages(
       final ServerServiceDefinition serviceDef,
       final MethodDescriptor.Marshaller<T> marshaller) {
-    final ServerServiceDefinition.Builder serviceBuilder = ServerServiceDefinition
-        .builder(serviceDef.getName());
-    for (final ServerMethodDefinition<?, ?> definition : serviceDef.getMethods()) {
-      final MethodDescriptor<?, ?> originalMethodDescriptor = definition.getMethodDescriptor();
-      final MethodDescriptor<T, T> wrappedMethodDescriptor = MethodDescriptor
-          .create(originalMethodDescriptor.getType(),
-              originalMethodDescriptor.getFullMethodName(),
+    List<MethodDescriptor> wrappedMethods = new ArrayList<MethodDescriptor>(
```

Comment:
Is it allowed style to use raw types? I'd think `List<MethodDescriptor<?, ?>>` would be better since Java's type system can do weird things when generic types appear as raw.